src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES = new Properties();
+
+    static {
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Root1234");
```

Comment:
Storing credentials in code is a security risk. Consider using environment variables or a configuration file to keep sensitive information out of the source code.